### PR TITLE
Add account access token support

### PR DESCRIPTION
**Pull request by zdavis-rollbar**
## What

One account token can now drive every tool: config accepts `ROLLBAR_ACCOUNT_ACCESS_TOKEN` (or `accountToken` in `.rollbar-mcp.json`, including an account-only config with no per-project tokens at all), projects are discovered and cached from `GET /projects`, and requests get `project_id` injected per endpoint — query param for GETs, JSON body for the `PATCH` in update-item, and repeated `project_ids=` query params for item search. Project-token configs are unchanged, and a legacy `ROLLBAR_ACCESS_TOKEN` is lazily probed once to detect whether it's actually an account token.

## Why

Customers with prompts that span multiple projects currently have to collect and configure a token per project. One account token, with per-call project selection, removes that friction and matches how account-scoped tokens actually work on the Rollbar API side.

## Details worth knowing

- The auth header is identical for both token types (`X-Rollbar-Access-Token`); project scoping happens entirely through request params, which differ by endpoint — this is why the param-building is centralized in one helper and regression-tested, especially for `GET /items/`, which only accepts *repeated* `project_ids=` params (a singular `project_id` or comma-joined list hangs ~25s then 422 in production).
- A second review pass (thanks to an independent pass from Codex) turned up several real edge cases that got fixed here too, each with its own regression test:
  - Account-token-only config files (`{ "accountToken": "..." }` alone, no `projects`/`token`) were rejected by the schema — the documented example didn't actually work.
  - `delivery="resource"` on `get-replay` could hand back a `rollbar://` link that was guaranteed to fail on read, in multi-project account-token mode — the guard was checking the wrong thing.
  - A hybrid config (explicit per-project tokens plus an account token) couldn't reach any project beyond the explicitly-listed ones, because the `project` parameter's schema was a static enum that never considered the account token.
  - A legacy `ROLLBAR_ACCESS_TOKEN` that probes as a plain project token was silently ignoring a mismatched `project` argument instead of rejecting it — a request for the wrong project name would silently get served by whatever token was configured.
- README updated to match: real token-creation steps, an account-token client example, and corrected scope guidance (`update-item` needs both read and write in account mode, since resolving the target project via `GET /projects` always happens before the `PATCH`).

## Testing

- `npm test` — 248/248 passing
- `npm run test:coverage` — 97.27% statements / 89.3% branches / 98.61% functions / 97.17% lines, above the configured 80/80/90/80 thresholds
- `npm run lint` / `npm run format` — clean
- `npm run build` — clean

Linear: ROL-1016

### CODE REVIEW

File: src/utils/projects-cache.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  forceRefresh: boolean,
+): Promise<RollbarProject[]> {
+  const key = cacheKey(token, apiBase);
+  let entry = cache.get(key);
+
+  if (forceRefresh || !entry) {
+    const fetchPromise = fetchProjects(token, apiBase);
+    entry = { projects: entry?.projects ?? [], fetchPromise };
+    cache.set(key, entry);
+    const projects = await fetchPromise;
```
[the remark this answers is not in the document]
Comment (waltjones) [reply src/utils/projects-cache.ts]: This looks significant since network errors are common at moxapi.

[the remark this answers is not in the document]
Comment (zdavis-rollbar) [reply src/utils/projects-cache.ts]: Fixed in latest push.  Now the cache entry clears on failure, so the next call retries against the API.

File: src/tools/get-replay.ts [hunk 2]
```diff
@@ -74,12 +74,19 @@ export function registerGetReplayTool(server: McpServer) {
...
 
-      if (deliveryMode === "resource" && PROJECTS.length > 1) {
-        throw new Error(
-          'delivery="resource" is not supported when multiple projects are configured. Use delivery="file" and specify the project parameter instead.',
-        );
+      if (deliveryMode === "resource") {
+        const accountMode = await getAccountModeInfo();
+        const tooManyProjects = accountMode.active
+          ? (accountMode.enabledProjectCount ?? 0) > 1
+          : PROJECTS.length > 1;
```
[the remark this answers is not in the document]
Comment (zdavis-rollbar) [reply src/tools/get-replay.ts:88]: Fixed in latest push.  It now sums both.

**Review by waltjones — APPROVED** [review]
